themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -28,36 +26,45 @@
     <label><?=$this->translate('Sort') ?></label>
     <div class="btn-group">
       <?php foreach ($this->sortOptions as $key => $sort): ?>
-        <a href="<?=$urlBase . '&amp;facetpage=1&amp;facetsort=' . urlencode($key) ?>" class="btn btn-default js-facet-sort<?php if ($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>" data-lightbox-ignore><?=$this->translate($sort) ?></a>
+        <a href="<?=$urlBase . '&facetpage=1&facetsort=' . urlencode($key) . '&contains=' . urlencode($contains) ?>" class="btn btn-default js-facet-sort<?php if($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>"><?=$this->translate($sort) ?></a>
```

Comment:
Since $urlBase is no longer being HTML escaped (which makes sense, due to the fact that you're passing it to AJAX through JSON later in the template), you should probably add a `$this->escapeHtmlAttr()` around HTML attributes that make use of it. I think it's still a good idea to properly escape the HTML even though I agree that keeping the URL variable unescaped makes sense for greater flexibility across use cases.